src/ui-components/Autocomplete/Autocomplete.tsx

```diff
@@ -24,6 +28,9 @@ export const Autocomplete = <T extends IAutoCompleteItem>({
   inputClasses,
   error,
   onValueChange,
+  onChange,
```

Comment:
Don't put it here it will break other logics

Reply:
done